### PR TITLE
BAN-1163: enable disable wayland

### PR DESCRIPTION
**Pull request by asinghjc**
## Issues
* [BAN-1163](https://jumpcloud.atlassian.net/browse/BAN-1163) -  enable disable wayland windowing system

## What does this solve?
This script enables or disables wayland windowing system.
Remote-assist application does not work with wayland windowing system.
windowing system will need to be switched to x11.

## Screenshots

[BAN-1163]: https://jumpcloud.atlassian.net/browse/BAN-1163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: scripts/linux/wayland_enable.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+################################################################################
+
+if [[ "${UID}" != 0 ]]; then
+    (>&2 echo "Error:  $0 must be run as root")
+    exit 1
+fi
+
+enable_wayland() {
+    case ${linuxType} in
+        ubuntu|\"rocky\"|\"centos\")
```
Comment (spairajc) [inline scripts/linux/wayland_enable.sh:13]: why we differentiating between these and debian ?

File: scripts/linux/wayland_enable.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            `sed -i 's/WaylandEnable=.*/#WaylandEnable=false/' /etc/gdm3/custom.conf`
+            ;;
+        debian)
+            `sed -i 's/WaylandEnable=.*/#WaylandEnable=false/' /etc/gdm3/daemon.conf`
```
Comment (spairajc) [inline scripts/linux/wayland_enable.sh:17]: Shouldn't this here be true ?

File: scripts/linux/wayland_enable.sh (hunk 1 above)
Comment (asinghjc) [reply scripts/linux/wayland_enable.sh:13]: the files in which we need to make the change are different for ubuntu|\"rocky\"|\"centos\" (/etc/gdm3/**_custom.conf_**) and debian (/etc/gdm3/**_daemon.conf_**).

File: scripts/linux/wayland_enable.sh (hunk 2 above)
Comment (asinghjc) [reply scripts/linux/wayland_enable.sh:17]: Not sure, the comment given in the conf file says that the line **_WaylandEnable=false_** need be commented and uncommented for the wayland enabling and disabling.

**Review by spairajc — APPROVED** [review]

**Pull request merged**